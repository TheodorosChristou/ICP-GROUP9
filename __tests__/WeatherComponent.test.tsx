import React from 'react';
import { render } from '@testing-library/react';
import WeatherPage from '@/pages/weatherpage';

describe('WeatherPage', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<WeatherPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
