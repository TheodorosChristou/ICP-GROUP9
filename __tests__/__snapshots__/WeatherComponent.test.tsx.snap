// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherPage matches snapshot 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
